### PR TITLE
Add EMQX enhanced app

### PR DESCRIPTION
**Pull request by techthom95**

### CODE REVIEW

File: EMQX/EMQX.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        echo $test->status;
+    }
+
+    public function livestats()
+    {
+        $status = 'inactive';
+        $res = parent::execute($this->url('status'));
+        $details = json_decode($res->getBody());
+
+        $data = [];
```
Comment (keriati) [inline EMQX/EMQX.php:26]: Still work in progress?

Comment (techthom95) [reply EMQX/EMQX.php:26]: No clue, just requested it from the heimdall site (https://apps.heimdall.site/request/list), following the request manual ...

Comment (keriati) [reply EMQX/EMQX.php:26]: OK, I see. Somebody would need to also write the code for this, it's just the template so far.

**Comment by mvdkleijn** [comment]
In an effort to cleanup, I'm going to close this PR.

The template people can download from https://apps.heimdall.site/request/list is purely for developers wanting to get a quick start at implementing a new app.

It does **not** magically create an (enhanced) app, a dev still needs to implement stuff for that.

If you just want to get the app available as a simple icon+link (foundation app), you could download the foundation zip, add an appropriate image based on the [image requirements](https://github.com/linuxserver/Heimdall-Apps/wiki/Image-requirements) and create a PR.

**Pull request closed without merging**